benchmarks/high_volume_reads_test.go

```diff
@@ -17,126 +18,100 @@ import (
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-
-					// Querier
-					job = benchCfg.Metrics.Jobs.Querier
-					annotation = metrics.QuerierAnnotation
-
-					err = metricsClient.Measure(e, metrics.ContainerCPU(job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-
-					if benchCfg.Metrics.EnableCadvisorMetrics {
-						err = metricsClient.Measure(e, metrics.ContainerMemoryWorkingSetBytes(job, samplingRange, annotation))
-						Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					}
-
-					err = metricsClient.Measure(e, metrics.RequestReadsQPS(job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					err = metricsClient.Measure(e, metrics.RequestDurationOkQueryRangeAvg(job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					err = metricsClient.Measure(e, metrics.RequestDurationOkQueryRangePercentile(99, job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					err = metricsClient.Measure(e, metrics.RequestDurationOkQueryRangePercentile(50, job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					err = metricsClient.Measure(e, metrics.RequestQueryRangeThroughput(job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-
-					// Ingesters
-					job = benchCfg.Metrics.Jobs.Ingester
-					annotation = metrics.IngesterAnnotation
-
-					err = metricsClient.Measure(e, metrics.ContainerCPU(job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-
-					if benchCfg.Metrics.EnableCadvisorMetrics {
-						err = metricsClient.Measure(e, metrics.ContainerMemoryWorkingSetBytes(job, samplingRange, annotation))
-						Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					}
-
-					err = metricsClient.Measure(e, metrics.RequestReadsGrpcQPS(job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					err = metricsClient.Measure(e, metrics.RequestDurationOkGrpcQuerySampleAvg(job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					err = metricsClient.Measure(e, metrics.RequestDurationOkGrpcQuerySamplePercentile(99, job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-					err = metricsClient.Measure(e, metrics.RequestDurationOkGrpcQuerySamplePercentile(50, job, samplingRange, annotation))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-
-					err = metricsClient.Measure(e, metrics.RequestBoltDBShipperReadsQPS(job, samplingRange))
-					Expect(err).Should(Succeed(), fmt.Sprintf("Failed - %v", err))
-				}, samplingCfg)
-			})
+		It("should measure metrics", func() {
```

Comment:
Can we give the `It()` blocks more useful descriptions. FWIW if one concatenates the blocks as of now we get:
`High Volume Reads Querying logs from Loki service Configuration: %s should measure metrics`

This is not a human-readable description for this test. I fully understand that we have now two suites `high volume reads` and `high volume writes` that we can parameterize for different scenarios (e.g. ingestion volumes, query ranges). But executing the suites with given parameters should emit meaningful descriptions per run for the tester.